src/main/kotlin/org/sert2521/infiniterecharge2020/climber/ClimberCommands.kt

```diff
@@ -0,0 +1,31 @@
+package org.sert2521.infiniterecharge2020.climber
+
+import edu.wpi.first.wpilibj.DigitalInput
+import org.sert2521.sertain.coroutines.periodic
+import org.sert2521.sertain.events.onTick
+import org.sert2521.sertain.subsystems.doTask
+import org.sert2521.sertain.subsystems.use
+
+suspend fun climberUp() = doTask {
+    val climber = use<Climber>()
+    action {
+        onTick {
+            if (!DigitalInput(0).get()) {
```

Comment:
Usually we like to put sensor statuses in the Subsystem class with something like `val climberAtTop get() = DigitalInput(-1)` and then check that value in the command with `climberAtTop`